thyrel-api/thyrel-api/Models/Sentence.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace thyrel_api.Models
+{
+    public class Sentence
+    {
+        public int ID { get; set; }
```

Comment:
```suggestion
        public int Id { get; set; }
```
Format `Id` à changé partout